rust-toolchain.toml

```diff
@@ -1,5 +0,0 @@
-[toolchain]
```

Comment:
Since the toolchain file helps us to develop in similar environments, I'd [override](https://rust-lang.github.io/rustup/overrides.html) instead of remove

Reply:
I agree, this is what we're doing locally. I removed it during development, since we were tinkering with the environment image. We can reintroduce this file later in a separate PR.